src/vmware/HISTORY.md

```diff
@@ -1,5 +1,22 @@
 # Release History
 
+## 5.0.0 (2022-10)
+
+- Update to AVS 2022-05-01 API
+- Add `az vmware cluster list-zones`
+- Add `az vmware workload-network show`
```

Comment:
Please remove both workload-network show & list. The operations were only added to satisfy the ARM resource hierarchy.

Reply:
Removed commands